src/Enum.php

```diff
@@ -569,9 +569,9 @@ public function toArray()
     /**
      * Transform the enum when it's passed through json_encode.
      *
-     * @return string
+     * @return mixed
      */
-    public function jsonSerialize()
+    public function jsonSerialize(): mixed
```

Comment:
```suggestion
    public function jsonSerialize()
```

`mixed` is only available from PHP 8.0 forward.